src/dcos_e2e/backends/_base_classes.py

```diff
@@ -48,6 +38,28 @@ def __init__(
         """
 
     @abc.abstractmethod
+    def install_dcos_from_url(
+        self,
+        build_artifact: str,
+        extra_config: Dict[str, Any],
+        log_output_live: bool,
+    ) -> None:
+        """
+        Stub for the DC/OS advanced installation method.
```

Comment:
Instead of this, give the docstring for what the method should do. For example, `destroy` has:

```
Destroy all nodes in the cluster.
```